extensions/typescript-language-features/package.nls.json

```diff
@@ -92,6 +92,7 @@
 	"typescript.updateImportsOnFileMove.enabled.always": "Always update paths automatically.",
 	"typescript.updateImportsOnFileMove.enabled.never": "Never rename paths and don't prompt.",
 	"typescript.autoClosingTags": "Enable/disable automatic closing of JSX tags.",
+	"typescript.fold.compact": "Enabled/disable compact fold.",
```

Comment:
To help users discover this feature, add a description of what compact folding does. For example: `Enabled/disable compact folding. This hides the closing } of folded blocks.`